litebird_sim/observations.py

```diff
@@ -551,4 +557,96 @@ def get_times(self, normalize=False, astropy_times=False):
         else:
             t0 = self.start_time
 
-        return t0 + np.arange(self.n_samples) / self.sampling_rate_hz
\ No newline at end of file
+        return t0 + np.arange(self.n_samples) / self.sampling_rate_hz
+
+    # Deprecated methods: Remove ASAP >>>
+    def get_quaternion_buffer_shape(self, num_of_detectors=None):
+        """Deprecated: see scanning.get_quaternion_buffer_shape
+        """
+        logging.warn(
+            "Observation.get_quaternion_buffer_shape is deprecated and will be "
+            "removed soon, user scanning.get_quaternion_buffer_shape instead")
```

Comment:
"user" -> "use" (the same typo happens below a few times)

Reply:
Fixing, thanks